README.md

```diff
@@ -87,14 +87,35 @@ and write the result to a local directory.
 
 ## One-time CLI installation
 
-The quick answer is to just run
-`go install github.com/abcxyz/abc/cmd/abc@latest` .
+There are two ways to install:
```

Comment:
Should this section just be called `Installation`, people will want to update this and one-time makes it sound like it auto updates or something?